test/SILGen/subscript_accessor.swift

```diff
@@ -11,3 +11,9 @@ struct X<T> {
     set { }
   }
 }
+
+  // CHECK: sil{{.*}}S18subscript_accessor9testXRead1xxAA1XVyxG_tlF
+@_specialize(where T == (Int, Int))
```

Comment:
If you want to test the @_specialize attribute you need to use -emit-sil -O

Reply:
Good catch; fix coming in the next PR